SyntaxKit/AttributedParser.swift

```diff
@@ -43,6 +43,16 @@ open class AttributedParser: Parser {
         }
     }
 
```

Comment:
Maybe add a documentation comment that only the foreground attributes are changed